go-controller/pkg/ovn/gress_policy.go

```diff
@@ -12,9 +12,16 @@ import (
 )
 
 type gressPolicy struct {
+	namespace  string
+	name       string
 	policyType knet.PolicyType
 	idx        int
 
+	// portGroupUUID is the UUID of the parent NetworkPolicy port group
+	portGroupUUID string
+	// portGroupName is the name of the parent NetworkPolicy port group
+	portGroupName string
```

Comment:
These are only used by `localPodAddACL` and it might make more sense to pass them in there?

If they are kept as gressPolicy-level fields, their name should reflect the fact that they are namespacePolicy-level data. `policyPortGroupName` or `npPortGroupName`, etc

(Likewise maybe `policyName` instead of `name` above)

Reply:
Done.